src/System.Collections/tests/Resources/System.Collections.Tests.rd.xml

```diff
@@ -1,7 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
 <Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
-  <Library>
+  <Library Name="System.Collections.Tests">
+    <Assembly Name="*Application*" DoNotInline="true" DoNotOptimize="true" />
```

Comment:
Thanks Santi for the info! Don't bother with it. I'm submitting a commit for it now.